blazor/rich-text-editor/undo-redo-manager.md

```diff
@@ -9,14 +9,14 @@ documentation: ug
 
 # Undo Redo Manager in Blazor Rich Text Editor
 
-Undo and redo tools allows to edit the text by disregard or cancel the recently made changes and restore it to previous state. It is a useful tool to restore the performed action which got changed by mistake. By default, upto 30 actions can be undo/redo in the editor.
+The undo and redo tools allow you to edit the text by disregarding or cancelling the recently made changes and restoring it to its previous state. It is a useful tool for restoring a previously performed action that was accidentally changed. In the editor, you can undo or redo up to 30 actions by default. 
 
 To undo and redo operations, do one of the following:
 
 * Press the undo/redo button on the toolbar
 * Press the Ctrl + Z/Ctrl + Y combination on the keyboard
```

Comment:
use **kbd** tag to highlight the key combination like <kbd>Ctrl + Z</kbd> / <kbd>Ctrl + Y</kbd>